tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature

```diff
@@ -0,0 +1,27 @@
+Feature: checking file versions using file id
+  As a user
+  I want to share file outside of the space
+  So that other users can access the file
+
+
+  Scenario: check the file versions of a shared file
```

Comment:
You can extend this scenario to have coverage for different share role: See https://github.com/owncloud/ocis/issues/7824#issue-2013952774